pkg/reconciler/testing/table.go

```diff
@@ -55,7 +55,8 @@ type TableRow struct {
 	// WantUpdates holds the set of Update calls we expect during reconciliation.
 	WantUpdates []clientgotesting.UpdateActionImpl
 
-	// WantUpdates holds the set of Update calls we expect during reconciliation.
+	// WantStatusUpdates holds the set of Update calls, with dprotaso`status` subresource set,
```

Comment:
Looks like a copy paste error?

```suggestion
	// WantStatusUpdates holds the set of Update calls, with `status` subresource set,
```